index.html

```diff
@@ -557,24 +566,47 @@ <h3>Terminology</h3>
 
           <dt><dfn class="export" data-lt="controller(s)|Controllers">controller</dfn></dt>
           <dd>
-An entity that is [=authorized=] to perform an action with a specific resource,
-such as update a [=controller document=] or use a cryptographic key to generate
-a digital signature.
+            <p>
+An entity that is capable of performing an action with a specific resource,
+such as updating a [=controller document=] or generating a [=proof=] using a [=verification method=].
+            </p>
           </dd>
 
           <dt><dfn class="export">controller document</dfn></dt>
           <dd>
-A set of data that specifies one or more relationships between a
-[=controller=] and a set of data, such as a set of public cryptographic keys.
+            <p>
+A document that contains cryptographic material and lists service
+endpoints that can be used for verifying proofs from, and interacting
+with, the [=controller=] of an identifier.
+            </p>
           </dd>
 
           <dt><dfn data-lt="subjects">subject</dfn></dt>
           <dd>
-The entity identified by the `id` property in a [=controller document=].
-Anything can be a subject: person, group, organization, physical thing, digital
-thing, logical thing, etc.
+            <p>
+An entity that is referred to by the `id` property in a [=controller document=]
+when it is used as a subject in other contexts, such as authentication or
+attestations, e.g., Verifiable Credentials. Anything can be a subject: 
+person, group, organization, physical thing, digital thing, logical thing, etc. 
+            </p>
+            <p class="note">
+It is expected that the subject referred to by the id of a controller
+document be consistent over time, so that a set of proofs with the
+same identifier as subject can be interpreted as referring to the same
+entity. For example, it is preferred that credential issuers require
+subjects to demonstrate proof of control over their identifiers before
+issuing a credential with that subject, creating assurances that the same
+entity was involved as the subject in the issuing of each credential.
+However, there are valid cases where that is either impossible or
+unreasonable. For example, when a parent requests a Verifiable Credential
+for their child. There are also invalid cases where an issuer simply makes
+a mistake or intentionally issues a false statement. All of these
+possibilities should be considered when evaluating the security of relying
+on a given identifier for any purpose. See the section on Identifier  Ambiguity in Security Considerations.
+            </p>
```

Comment:
The "see the section" at the end should be a cite/link.
```suggestion
            <div class="note">
              <p>
It is expected that the [=subject=] referred to by the `id` of a
[=controller document=] will be consistent over time, so that a set of
proofs with the same [=identifier=] as [=subject=] can be interpreted
as referring to the same entity. For example, it is preferred that
credential issuers require that [=subjects=] demonstrate proof of
control over their [=identifier=] before issuing a credential with that
[=identifier=] as subject, creating assurance that the same entity was
involved in the issuance of each credential with that [=identifier=]
as [=subject=].
              </p>
              <p>
However, there are valid cases where that practice is either impossible or
unreasonable; for example, when a parent requests a Verifiable Credential
for their child. There are also cases where an issuer simply makes a mistake
or intentionally issues a false statement. All of these possibilities should
be considered when evaluating the security impacts of reliance on a given
[=identifier=] for any given purpose. See the section on Identifier Ambiguity
in Security Considerations.
              </p>
            </div>
```